Description: add DefaultRuntimeDir in test apache config
 The tests in ModPerl-Registry/ and Apache-Reload/ need DefaultRuntimeDir
 set, otherwise mod_auth_digest tries to write to /var/run and fails.
Origin: vendor
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2013-06-03

--- a/ModPerl-Registry/t/conf/extra.conf.in
+++ b/ModPerl-Registry/t/conf/extra.conf.in
@@ -1,5 +1,7 @@
 #this file will be Include-d by @ServerRoot@/httpd.conf
 
+DefaultRuntimeDir run/
+
 # Adjust PerlInterpMax and PerlInterpMaxSpare if the requirements change
 # at the moment all tests require only 1 interprter
 <IfDefine PERL_USEITHREADS>
--- a/Apache-Reload/t/conf/extra.conf.in
+++ b/Apache-Reload/t/conf/extra.conf.in
@@ -1,3 +1,5 @@
+DefaultRuntimeDir run/
+
 PerlSwitches -I@ServerRoot@/../../Apache-Test/lib   \
              -I@ServerRoot@/../../blib/lib          \
              -I@ServerRoot@/../../blib/arch
